tests/testdata/has_external.css

```diff
@@ -0,0 +1,3 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import './does_not_exist.css';
+@import './b.css';
```

Comment:
This case can be bundled without any semantics change, but for the following case, it's difficult to keep the semantics.
```css
@import './b.css'; // bundled
@import "https://fonts.googleapis.com/css2?family=Roboto&display=swap"; // externalized
```
related: https://github.com/postcss/postcss-import/issues/536, https://github.com/parcel-bundler/parcel/issues/5840#issuecomment-778857252, https://github.com/evanw/esbuild/issues/465#issuecomment-787048779

Maybe it's fine to simply change the semantics in this case? (both [postcss-import](https://github.com/postcss/postcss-import/issues/536#:~:text=I%20suggest%20we%20close%20this%20issue%20as%20won%27t%20fix) and [esbuild](https://github.com/evanw/esbuild/pull/1293#:~:text=Note%20that%20while,match%20the%20browser.) seems to do that)